lib/fixture_factory/errors.rb

```diff
@@ -30,11 +30,12 @@ def initialize(method_name)
   end
 
   class WrongClassError < Error
-    def initialize(class_name)
+    def initialize(klass)
+      location, line_number = klass.source_location
       super(
         <<~MSG.squish
-          No class named "#{class_name}".
-          Try using the `class_name` option in your definition to specify a valid class name.
+          Constant defined in file #{location} on line #{line_number} is not defined.
```

Comment:
The location would be inside the gem if the `class_name` option is used, so this error may not be very helpful in that case. If `class_name` isn't in deprecation cycle can we treat it like a first class feature and switch on string or proc for this error message?

Reply:
Your reasoning makes sense to me. I think this will add some complexity but I will give it a stab 👍